CONTRIBUTING.md

```diff
@@ -281,7 +258,7 @@ Using [Sphinx](http://www.sphinx-doc.org/en/stable/) the documentation in `docs`
             # Differrent weights:
             weights = [
                 1.0,
-                2.7,
+                2.6,
```

Comment:
I don't understand this change

Reply:
That was just because I was grepping "2.7" :D
Happy to undo it although it won't affect anything

Reply:
Ha, I wondered if it was this ;)

Might be useful to have an unofficial rule about using constants 3.x to make it easier when we get rid of old versions.